shell/platform/tizen/channels/window_channel.cc

```diff
@@ -43,10 +43,7 @@ void WindowChannel::HandleMethodCall(
     map[EncodableValue("height")] = EncodableValue(geometry.height);
     result->Success(EncodableValue(map));
   } else if (method_name == "setWindowGeometry") {
-#ifdef TIZEN_RENDERER_EVAS_GL
-    FT_LOG(Error) << "setWindowGeometry is not supported on Evas GL.";
-    result->NotImplemented();
-#else
+#ifndef WEARABLE_PROFILE
```

Comment:
`#ifndef WEARABLE_PROFILE` doesn't make sense here.

- Is resizing still not supported on Evas GL?
- Even if it's not supported, the current profile has nothing to do with the renderer type.
- `TizenWindowElementary::SetGeometry` should be no-op and print an error if resizing is not supported.
- If you really want to check the renderer type before performing the resize, you should keep a reference to `renderer` and use `renderer->type()`.

Reply:
I will more modify this part. please wait a minute.

Reply:
The window channel setup is done before CreateSurface.
I tried to make the channel setup a separate function, create a Renderer and pass it to the window channel, but I thought it was an unnecessary modification.
So I handled it by returning false from SetGeometry. I hope this is appropriate.

Reply:
Thanks. That looks great.